README.md

```diff
@@ -277,54 +271,71 @@ load_logs(new_optimizer, logs=["./logs.log"]);
-
-* With PyPI (pip):
-
-      pip install bayesian-optimization
+This introduction covered the most basic functionality of the package. Checkout the [basic-tour](http://bayesian-optimization.github.io/BayesianOptimization/basic-tour.html) and [advanced-tour](http://bayesian-optimization.github.io/BayesianOptimization/advanced-tour.html), where you will find detailed explanations and other more advanced functionality. Also, browse the [examples](http://bayesian-optimization.github.io/BayesianOptimization/examples.html) for implementation tips and ideas.
 
-* With conda (from conda-forge channel):
-
-      conda install -c conda-forge bayesian-optimization
-
-The bleeding edge version can be installed with:
-
-    pip install git+https://github.com/fmfn/BayesianOptimization.git
+## Installing from GitHub
+To install directly from master, simply run
+```
+pip install https://github.com/bayesian-optimization/BayesianOptimization/archive/master.zip
+```
+Please note that these builds are not necessarily stable.
 
-If you prefer, you can clone it and run the setup.py file. Use the following
-commands to get a copy from Github and install all dependencies:
+If you prefer, you can also clone the repository and run `setup.py`:
+```
+git clone https://github.com/bayesian-optimization/BayesianOptimization.git
+cd BayesianOptimization
+python setup.py install
+```
+    
 
-    git clone https://github.com/fmfn/BayesianOptimization.git
-    cd BayesianOptimization
-    python setup.py install
+## Minutiae
 
-Citation
-============
+### Citation
 
-If you used this package in your research and is interested in citing it here's how you do it:
+If you used this package in your research, please cite it:
 
 ```
 @Misc{,
     author = {Fernando Nogueira},
     title = {{Bayesian Optimization}: Open source constrained global optimization tool for {Python}},
     year = {2014--},
-    url = " https://github.com/fmfn/BayesianOptimization"
+    url = " https://github.com/bayesian-optimization/BayesianOptimization"
+}
+```
+If you used any of the advanced functionalities, please additionally cite the corresponding publication:
+
+For the `SequentialDomainTransformer`:
+```
+@article{
+    author = {Stander, Nielen and Craig, Kenneth},
+    year = {2002},
+    month = {06},
+    pages = {},
+    title = {On the robustness of a simple domain reduction scheme for simulation-based optimization},
+    volume = {19},
+    journal = {International Journal for Computer-Aided Engineering and Software (Eng. Comput.)},
+    doi = {10.1108/02644400210430190}
+}
+```
+
+For constrained optimization:
+```
+@inproceedings{gardner2014bayesian,
+    title={Bayesian optimization with inequality constraints.},
+    author={Gardner, Jacob R and Kusner, Matt J and Xu, Zhixiang Eddie and Weinberger, Kilian Q and Cunningham, John P},
+    booktitle={ICML},
+    volume={2014},
+    pages={937--945},
+    year={2014}
 }
 ```
 
-# Dependencies
+### Dependencies
```

Comment:
suggest we remove this - it should be clear from the packaging tools. it is even now, and will be more so when we switch to poetry

Reply:
done.